uweb3/templateparser.py

```diff
@@ -550,7 +550,7 @@ def __init__(self, template_path, parser=None):
     try:
       self._file_name = os.path.abspath(template_path)
       self._file_mtime = os.path.getmtime(self._file_name)
-      with open(self._file_name) as templatefile:
+      with open(self._file_name, encoding='utf-8') as templatefile:
```

Comment:
Move 'utf-8' to a variable templateencoding on the templateparser class and use that. This would allow developers to set different encodings for reading template files.

Reply:
Updated the code to use the property templateEncoding from the parser which is provided by a parameter in the constructor of the Parser. However the Parser class is instantiated in the Base class of the PageMaker, so there is still no way to supply the Parser with this parameter without changing the underlying uweb3 code yourself.